core/os/keyboard.h

```diff
@@ -260,6 +260,12 @@ enum class KeyModifierMask {
 	GROUP_SWITCH = (1 << 30)
 };
 
+enum class KeyLocation {
+	UNKNOWN,
```

Comment:
As suggested in the other thread, I would also rename the C++ stuff to `ANY` / `any` wherever `UNKNOWN` / `unknown` is currently used. It's better to be consistent with the user facing name, and IMO it's a better name overall anyway.

Reply:
Well let's settle the discussion in the other thread first before doing further changes, to make sure we have a consensus on what this should be.